test/golden-utils.js

```diff
@@ -20,6 +20,7 @@ const Diff = require('text-diff');
 const PNG = require('pngjs').PNG;
 const jpeg = require('jpeg-js');
 const pixelmatch = require('pixelmatch');
+const c = require('colors/safe');
```

Comment:
nit: s/c/colors
You taught me that we don't abbreviate like that :)

Reply:
done